### PR TITLE
Added pypi-publish workflow template

### PR DESCRIPTION
**Pull request by Jawayria**
Added a workflow template to be used for pypi package releases in edx repositories

### CODE REVIEW

File: workflow-templates/pypi-publish.properties.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "PYPI Package release Workflow",
+    "description": "Use this workflow to publish pypi package release",
```
Comment (mraarif) [inline workflow-templates/pypi-publish.properties.json:3]: shouldn't this be 'to publish your package to PyPI?

File: workflow-templates/pypi-publish.properties.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "PYPI Package release Workflow",
```
Comment (mraarif) [inline workflow-templates/pypi-publish.properties.json:2]: this should be PyPI

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
+#ignore .idea directory
+.idea
```
Comment (mraarif) [inline .gitignore:3]: should end with a new line

File: workflow-templates/pypi-publish.properties.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    "description": "Use this workflow to publish pypi package release",
+    "iconName": "edx-workflow-template-icon",
+    "categories": [
+        "python"
+    ],
+    "filePatterns": [
+        "setup.py$"
+    ]
+}
```
Comment (mraarif) [inline workflow-templates/pypi-publish.properties.json:11]: same here

**Review by mraarif — APPROVED** [review]

**Review by mzulqarnain1 — APPROVED** [review]

File: workflow-templates/pypi-publish.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+name: Publish package to PyPi
```
Comment (jmbowman) [inline workflow-templates/pypi-publish.yml:1]: Capitalization: "PyPi" -> "PyPI"

File: workflow-templates/pypi-publish.yml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        with:
+          python-version: 3.8
+
+      - name: Install pip
+        run: pip install -r requirements/pip.txt
+
+      - name: Build package
+        run: python setup.py sdist bdist_wheel
+
+      - name: Publish to PyPi
```
Comment (jmbowman) [inline workflow-templates/pypi-publish.yml:25]: Here also

**Review by jmbowman — APPROVED** [review]
Minor capitalization issues, otherwise looks good!

**Pull request merged**